.github/workflows/push.yml

```diff
@@ -0,0 +1,73 @@
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by it. Please use
+# './plugin-template --github pulp_npm' to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+---
+name: Pulp on Push
+on:
+  push:
```

Comment:
Is this the same set of pipelines as the standard CI?  We could just have it trigger on either PRs or push.

Reply:
almost the same, I removed the `lint` part

Reply:
It's not optimal but we could just run our lint tests against pushes. We do that now I believe.

Reply:
What is the benefit of having a second workflow w/o the lint?  Even if we don't really need it, it would be less to maintain and doesn't add much in terms of resource use.

Unless it's likely that these workflows would need to diverge in the future, in which case it makes sense to go ahead and separate them.

Reply:
I'll close this and update the standard workflow

Reply:
nightly and release are pretty similar, I think we can combine them